cmds/minigames/connect4.js

```diff
@@ -102,55 +159,112 @@ class connect4 extends two_p_Game{
                             */
                             break;
                         case 2:
-                            await rmsg.edit(client.EmbedMaker(msg, game.stringify(), client.colors[current_color]))
-                            await ymsg.edit(client.EmbedMaker(msg, game.stringify(), client.colors[current_color]))
-                            iuser.send(msg.author.username + client.language.commands.game[userlang].wins)
-                            msg.author.send(msg.author.username + client.language.commands.game[userlang].wins)
-                            redcollector.stop()
-                            yellowcollector.stop()
+                            await rmsg.edit(
+                                client.EmbedMaker(
+                                    msg,
+                                    game.stringify(),
+                                    client.colors[current_color]
+                                )
+                            );
+                            await ymsg.edit(
+                                client.EmbedMaker(
+                                    msg,
+                                    game.stringify(),
+                                    client.colors[current_color]
+                                )
+                            );
+                            iuser.send(msg.author.username + client.language.commands.game[userlang].wins);
+                            msg.author.send(msg.author.username + client.language.commands.game[userlang].wins);
+                            redcollector.stop();
+                            yellowcollector.stop();
                             break;
 
                         case 3:
-                            await rmsg.edit(client.EmbedMaker(msg, game.stringify(), client.colors.white))
-                            await ymsg.edit(client.EmbedMaker(msg, game.stringify(), client.colors/white))
-                            iuser.send(client.language.commands.game[userlang].draw)
-                            msg.author.send(client.language.commands.game[userlang].draw)
-                            redcollector.stop()
-                            yellowcollector.stop()
+                            await rmsg.edit(
+                                client.EmbedMaker(
+                                    msg,
+                                    game.stringify(),
+                                    client.colors.white
+                                )
+                            );
+                            await ymsg.edit(
+                                client.EmbedMaker(
+                                    msg,
+                                    game.stringify(),
+                                    // client.colors / white =============== I'm not sure what this "/" does so I assume it is a typo
```

Comment:
I think this is a typo here
```suggestion
                                    // client.colors / white
                                    client.colors.white
```